packages/postgres-database/src/simcore_postgres_database/migration/versions/ec845b05fa59_add_product_name_to_services_vendor_.py

```diff
@@ -0,0 +1,42 @@
+"""add product_name to services_vendor_secrets
+
+Revision ID: ec845b05fa59
+Revises: 5c62b190e124
+Create Date: 2023-10-18 09:11:24.187006+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ec845b05fa59"
+down_revision = "5c62b190e124"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "services_vendor_secrets",
+        sa.Column("product_name", sa.String(), server_default="osparc", nullable=False),
+    )
+    op.create_foreign_key(
+        "fk_services_name_products",
+        "services_vendor_secrets",
+        "products",
+        ["product_name"],
+        ["name"],
+        onupdate="CASCADE",
+        ondelete="CASCADE",
```

Comment:
> you model says otherwise, but the migration does not seem to modify the primary key. are you sure this is correct?

I'm a bit puzzled about this one.
What I want is for product_name to be part of the primary key.

Reply:
well then it means the migration is incorrect.

Reply:
Pleas have a look at the new migration code, thing it should be good.